code/modules/mob/living/carbon/human/species_types/skeletons.dm

```diff
@@ -5,7 +5,12 @@
 	say_mod = "rattles"
 	sexes = 0
 	meat = /obj/item/food/meat/slab/human/mutant/skeleton
-	species_traits = list(NOBLOOD, HAS_BONE, NOEYESPRITES)
+	species_traits = list(
```

Comment:
speaking about consistency when you make this var inconsistent with all other species_traits for other species...

Reply:
im going to gouge my own eyeballs out

Reply:
u were right the first time